packages/tag/tag.scss

```diff
@@ -4,19 +4,26 @@
 $_tag-background-color: jkl.$color-varde;
 $_tag-color: jkl.$color-granitt;
 
+$_dismiss-focus-border-color: jkl.$color-granitt;
+$_dismiss-hover-color: jkl.$color-stein;
+
 $_tag-background-color--success: colors.varslingsfarge("suksess");
 $_tag-background-color--error: colors.varslingsfarge("feil");
 $_tag-background-color--info: colors.varslingsfarge("info");
 $_tag-background-color--warning: colors.varslingsfarge("advarsel");
 
+$_text-input-button-width: jkl.rem(48px);
+$_text-input-height: jkl.rem(48px);
+
 .jkl-tag {
     @include jkl.text-style("heading-5");
     background-color: $_tag-background-color;
     border-radius: jkl.rem(4px);
     color: $_tag-color;
-    display: inline-block;
     padding: jkl.$spacing-3xs jkl.$spacing-xs;
     white-space: nowrap;
+    display: inline-flex;
+    align-items: center;
```

Comment:
Ser ikke helt grunnen til denne endringen vekk fra `inline-block`.

Reply:
Den var for å få alignet ikonet og teksten